swagger_parser/lib/src/generator/templates/dart_freezed_dto_template.dart

```diff
@@ -21,7 +21,7 @@ ${dartImports(imports: dataClass.imports)}
 part '${dataClass.name.toSnake}.freezed.dart';
 part '${dataClass.name.toSnake}.g.dart';
 
-${descriptionComment(dataClass.description)}@Freezed()
+${descriptionComment(dataClass.description)}@freezed
```

Comment:
Add back `@Freezed`, because it is customizable

Reply:
https://github.com/Carapacik/swagger_parser/pull/97/commits/8a990f7f235fd3317ec9711d621affa667449309